front-end-ui/build.gradle

```diff
@@ -0,0 +1,103 @@
+import org.ysb33r.gradle.nodejs.tasks.NpmTask
+
+apply plugin: 'war'
+apply plugin: 'liberty'
+apply plugin: 'org.ysb33r.nodejs.npm'
+
+description = 'Microservice :: Frontend UI'
+dependencies {
+    providedCompile project(':shared-keystore')
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version:'17.0.0.3'
```

Comment:
should bump to 17.0.0.4.  You could do that version range thing that says [17.0.0.4,) and you will always get the latest.